src/components/Select/SingleSelect.tsx

```diff
@@ -0,0 +1,107 @@
+import { useCallback, useEffect, useState } from "react";
```

Comment:
there's a lot of code duplication between single select and multi select. 

The two are basically the same component, the only thing that should change should be the select function. 


We can respect the html `select` element and pass a `multiple` prop to the same component

Reply:
I wanted to create it as a single item 
I thought that since we separated the input to search text and numberfield we needed to do the same for select
@fneves Would like your thoughts

Reply:
Even if we export it as a separate component, the implementation should be the same. I think we should reuse the multiple select to implement the single select, just changing the value selection function and the displayed input